src/Command/MergeAssetsAndVariationsFilesCommand.php

```diff
@@ -201,6 +220,15 @@ private function isHeaderValid(CsvReader $reader, $row)
     private function getAssetManagerFileHeaders(): array
     {
         $assetHeaders = $this->assetsReader->getHeaders();
+
+        if ($this->withCategories === self::NO) {
+            // Removes the "categories" field from headers
+            $index = array_search(self::CATEGORIES, $assetHeaders);
```

Comment:
I think there is a one-liner, immutable alternative with array_filter. But this works as well.